engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -1332,14 +1334,14 @@ class PlatformDispatcher {
   }
 
   // Called from the engine, via hooks.dart
-  void _dispatchSemanticsAction(int nodeId, int action, ByteData? args) {
+  void _dispatchSemanticsAction(int viewId, int nodeId, int action, ByteData? args) {
     _invoke1<SemanticsActionEvent>(
```

Comment:
No action needed in this pr since we already integrate viewId into SemanticsActionEvent.

However, after taking a closer look, it feels to me we should also move onSemanticsActionEvent to flutter view given we already move the update semantics to flutter view.

This way, both semanticsupdate and semanticsactionEvent don't need to be annontated with viewId.